ramls/routing-lists.raml

```diff
@@ -1,47 +1,88 @@
 #%RAML 1.0
-title: "RoutingList"
+title: Routing Lists
```

Comment:
module descriptor file also should be updated, see comment here
https://github.com/folio-org/mod-orders/pull/874#discussion_r1549516070